### PR TITLE
[CURATOR-390] Create better summary of what Apache Curator is/does

### PR DESCRIPTION
**Pull request by oza**

### CODE REVIEW

**Comment by oza** [comment]
Based on discussion on CURATOR-390, I created this PR as a modification against README file.

**Comment by Randgalt** [comment]
The website source should be changed too: `src/site/confluence/index.confluence`

**Comment by Randgalt** [comment]
NOTE: I just updated the wiki's example email for release announcements https://cwiki.apache.org/confluence/display/CURATOR/Example+Emails

**Comment by oza** [comment]
Thanks a lot for your point. Updated index.confluence.

File: src/site/confluence/index.confluence [hunk 1]
```diff
@@ -2,7 +2,10 @@ h1. Welcome to Apache Curator
 
 h2. What is Curator?
 
-Curator _n &#x02c8;kyoor&#x035d;&#x02cc;&#x0101;t&#x0259;r_: a keeper or custodian of a museum or other collection \- A ZooKeeper Keeper.
```
Comment (Randgalt) [inline src/site/confluence/index.confluence]: I'd like to keep this line please.

**Comment by oza** [comment]
Done :-) Could you check the sentence?

File: src/site/confluence/index.confluence [hunk 2]
```diff
@@ -4,6 +4,10 @@ h2. What is Curator?
 
 Curator _n &#x02c8;kyoor&#x035d;&#x02cc;&#x0101;t&#x0259;r_: a keeper or custodian of a museum or other collection \- A ZooKeeper Keeper.
 
+Then, what is Apache Curator? Apache Curator is a Java/JVM client library for [[Apache ZooKeeper|https://zookeeper.apache.org/]], a distributed coordination service.
```
Comment (Randgalt) [inline src/site/confluence/index.confluence]: "Then, what is Apache Curator?" is unnecessary IMO. I prefer the line without this.

**Comment by oza** [comment]
OK, updated.

**Pull request merged**